Benchmark/README.md

```diff
@@ -14,9 +14,53 @@ to run all the available benchmarks, or run-<CHECK ID> to run a specific one.
 
 Or simply run the following command to do it automatically:
 ```
-./run-benchmarks.sh
+./run-benchmarks.py
```

Comment:
We should list Python 3 as an optional dependency at the start of the `README`.

Reply:
Yep, done